collector/compile-benchmarks/unicode-normalization-0.1.24/.gitignore

```diff
@@ -1,5 +1,5 @@
 target
-Cargo.lock
+#Cargo.lock
```

Comment:
This isn't needed, we can just force add the file to our git index and it will be tracked then. But no harm in this either.

Reply:
Right, from the how-to-update docs I wasn't sure if it was necessary or if `-f` was enough.